### PR TITLE
Add the Hoppinpunten and Haltes external subsidies

### PR DESCRIPTION
**Pull request by Riadabd**
(Addresses DL-4930 and DL-5120) Adds two external subsidies: Hoppinpunten and Haltes.

This PR should be used in conjuction with the following frontend PR (https://github.com/lblod/frontend-loket/pull/310), which adds support for handling external subsidies.

To test:
* Log in as any gemeente
* Go to the subsidies and add a new subsidy
* In case testing before 29 May, add `?testMode=true` to the URL to display the subsidies

Once done with the above steps:
* Two new subsidies should show up: `Subsidie voor Hoppinpunten` and `Subsidie voor het toegankelijk aanleggen of herinrichten van haltes en de uitrusting van haltes`
* Both should have `Oproep` in `Periode`, with the secondary `29 mei - N.V.T` text below.
* `Aanvragen tot` should be empty.
* Clicking on `Start aanvraag` should take you to `https://subsidiesmobiliteitsbeleid.vlaanderen.be/#/` which then re-directs to `https://authenticatie.vlaanderen.be/stb/html/ssologin`.

Given that the subsidies don't have a specific start date (tickets says any date as soon as possible), we can change the dates and merge the PR the day we want to deploy a new loket version.

### CODE REVIEW

File: config/resources/master-subsidies-domain.lisp [hunk 1]
```diff
@@ -100,7 +100,8 @@
 
 (define-resource subsidy-application-flow-step ()
   :class (s-prefix "lblodSubsidie:ApplicationStep")
-  :properties `((:order :integer ,(s-prefix "qb:order")))
+  :properties `((:order :integer ,(s-prefix "qb:order"))
+                (:is-replaced-by :uri, (s-prefix "dct:isReplacedBy")))
```
[the remark this answers is not in the document]
Comment (Windvis) [reply config/resources/master-subsidies-domain.lisp:104]: I'm not a fan of boolean sounding names for things that aren't booleans. I would go with something like `external-process`,`external-link`, `external-process-link` or something similar that suits the context better.

**Review by Windvis — APPROVED** [review]

**Comment by Windvis** [comment]
Looks good, I think we can merge this if we add a changelog entry. Should we create a release of the frontend and bump it here?

**Comment by Riadabd** [comment]
Do we do a frontend release now or wait to see if anything comes up for the LPDC filtering bugs Kevin found (if they are bugs) and then do one release altogether?

**Comment by Windvis** [comment]
I think it makes sense to create a release already. That way we can deploy this to QA.

**Pull request merged**